config/7.10.1/common.yaml

```diff
@@ -727,6 +727,8 @@ servers:
                       value: "[[$.Auth.LDAP.BindCredential]]"
                     - name: AUTH_LDAP_JAAS_SECURITY_DOMAIN
                       value: "[[$.Auth.LDAP.JAASSecurityDomain]]"
+                    - name: AUTH_LDAP_LOGIN_MODULE
+                      value: "[[.Auth.LDAP.LoginModule]]"
```

Comment:
appears you're missing `$` on front of env var as the others have set

Reply:
thanks!